modules/cluster-management/src/test/java/org/apache/ignite/internal/cluster/management/ClusterStatePersistentSerializerTest.java

```diff
@@ -73,17 +75,19 @@ void serializationAndDeserializationWithNulls() {
 
     @Test
     void v1CanBeDeserialized() {
-        byte[] bytes = Base64.getDecoder().decode("Ae++QwMBYQFiAwFjAWQFMy4wLjAHY2x1c3Rlcp1Ct7dR35ELuoeboFbabrgHY29uZmlnAztFBahaoEfJtxGam"
-                + "Q6WXJNFRfruL76Bv254dP54iF6V");
+        byte[] bytes = Base64.getDecoder().decode("Ae++QwMCYgJhAwJkAmMGMy4wLjAIY2x1c3Rlcu/Nq5B4VjQSIUNlhwm63P4HY29uZmlnA+/Nq5B4VhI0IUN"
```

Comment:
Why change it?

Reply:
Base64 representation was changed because I had to break serialization format in a backward-incompatible way.

Doing this, I realized that random UUIDs create more problems as I had to also change asserts on each such change (maybe we'll have to change the format again before we release 3.0). Fixed UUIDs don't create this problem.